ignite/engine/events.py

```diff
@@ -90,8 +90,8 @@ def __call__(
         if (every is not None) and not (isinstance(every, numbers.Integral) and every > 0):
             raise ValueError("Argument every should be integer and greater than zero")
 
-        if (once is not None) and not (isinstance(once, numbers.Integral) and once > 0):
-            raise ValueError("Argument once should be integer and positive")
+        if (once is not None) and not (isinstance(once, numbers.Integral) and once > 0) and not (all(isinstance(ele, int) for ele in once)):
```

Comment:
We should check if once is number or a Sequence. If it is a sequence than it should contain positive integral number. You check wont raise any issue if `once=[]`.